packages/icons/web/v2/vr_24.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24">
```

Comment:
The color of this icon is different from the VR logo in the card schemes category. Is that intentional? I tried to google it, but "vr" is an ambiguous term 😅

<img width="128" alt="Screenshot 2022-05-24 at 06 43 30" src="https://user-images.githubusercontent.com/11017722/170026833-dbec2bdd-b060-49f4-a33a-72a9c1087e00.png">

Reply:
actually both colors are right. if you look at [their website](https://www.vr.com.br/) they have two greens combined (which looks super weird 🤷 )

the payment method - #00be28 
the card scheme - #00a313